### PR TITLE
Added Cloud Patches 1.0.4 release notes

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Adds the release notes for Magento Cloud Patches version 1.0.4.

- [x] Technical review
- [x] Editorial review

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/release-notes/cloud-tools.html
- https://devdocs.magento.com/cloud/release-notes/mcp-release-notes.html
- https://devdocs.magento.com/cloud/release-notes/project-patch.html

## Links to Magento source code

[Updates to Cloud Patches source code for version 1.0.4](https://github.com/magento/magento-cloud-patches/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Release%3A+1.0.4%22)

### CODE REVIEW

File: src/cloud/release-notes/mcp-release-notes.md [hunk 1]
```diff
@@ -16,6 +16,23 @@ The `{{site.data.var.mcp}}` package uses the following version sequence: `<major
 {:.bs-callout-info}
 See [Apply patches]({{site.baseurl}}/cloud/project/project-patch.html) to learn how to apply Magento patches and hot fixes to your {{site.data.var.ece}} project.
 
+## v1.0.4
+*Release date: May 18, 2020*<br/>
```
Comment (meker12) [inline src/cloud/release-notes/mcp-release-notes.md:20]:
```suggestion
*Release date: May 12, 2020*<br/>
```

File: src/cloud/project/project-patch.md [hunk 2]
```diff
@@ -64,6 +64,12 @@ To apply and test a custom patch:
 
 1. Copy the patch file to the `/m2-hotfixes` directory.
 
+1. Add, commit, and push your code changes:
+
+   ```bash
+   git add -A && git commit -m "Apply <patch filename> patch" && git push origin
```
Comment (hguthrie) [inline src/cloud/project/project-patch.md:70]: Sometimes this works, but sometimes it gives errors to string the commands together. And you have a var in the message. Probably best to keep that generic. "Apply patch"

File: src/cloud/project/project-patch.md [hunk 3]
```diff
@@ -64,6 +64,12 @@ To apply and test a custom patch:
 
 1. Copy the patch file to the `/m2-hotfixes` directory.
 
+1. Add, commit, and push your code changes:
```
Comment (hguthrie) [inline src/cloud/project/project-patch.md:67]:
```suggestion
1. Add, commit, and push code changes.
```

File: src/cloud/release-notes/mcp-release-notes.md [hunk 4]
```diff
@@ -16,6 +16,23 @@ The `{{site.data.var.mcp}}` package uses the following version sequence: `<major
 {:.bs-callout-info}
 See [Apply patches]({{site.baseurl}}/cloud/project/project-patch.html) to learn how to apply Magento patches and hot fixes to your {{site.data.var.ece}} project.
 
+## v1.0.4
+*Release date: May 12, 2020*<br/>
```
Comment (hguthrie) [inline src/cloud/release-notes/mcp-release-notes.md:20]: Should have a blank line around headings I think.

File: src/cloud/release-notes/mcp-release-notes.md [hunk 5]
```diff
@@ -16,6 +16,23 @@ The `{{site.data.var.mcp}}` package uses the following version sequence: `<major
...
+
+-  **Amazon Pay checkout**—Fixes an issue with the Amazon Pay payment widget that prevented customers from changing the payment method on the _Review & Payments_ step during the checkout process.<!--MCLOUD-5930-->
+
+-  **Product display on Category pagew**—Fixes an issue that prevented products from displaying on the category page in _Show all pages_ view.<!--MCLOUD-5684-->
```
Comment (hguthrie) [inline src/cloud/release-notes/mcp-release-notes.md:24]: is `pagew` correct?

File: src/cloud/release-notes/mcp-release-notes.md [hunk 6]
```diff
@@ -16,6 +16,23 @@ The `{{site.data.var.mcp}}` package uses the following version sequence: `<major
...
+
+-  **Page Builder image upload**—Fixes a Page Builder interface issue that sometimes caused the following error when uploading images to the image gallery: `Destination folder is not writable or does not exist`<!--MCLOUD-5837-->
+
+-  **Suppress unnecessary Sitemap generation warnings**—Adds a retry attempt when errors occur during sitemap generation and skips customer email notification in cases where errors can be recovered automatically.<!--MCLOUD-3025-->
```
Comment (hguthrie) [inline src/cloud/release-notes/mcp-release-notes.md:28]: Should sitemap be capitalized?

**Comment by meker12** [comment]
running tests

**Pull request merged**